MetaMask/metamask-extension
# UX: Faster perceived performance of popup

## **Description**
With https://github.com/MetaMask/metamask-extension/pull/20843 we made JavaScript for the extension load asynchronously, which helped to get the popup displaying faster.  Unfortunately with a timeout of `0` to kick off the process, the popup still acts as though it's blocking (not async).  By adding a small timeout, the popup display *much faster*.

## **Manual testing steps**

1.  On current `develop`, mentally note how much time it takes for the popup to display
2. Check out this PR
3. Restart the extension
4. See that the popup displays *much* faster!

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
